lasagne/layers/recurrent.py

```diff
@@ -438,6 +468,24 @@ class Gate(object):
     nonlinearity : callable or None
         The nonlinearity that is applied to the input gate activation. If None
         is provided, no nonlinearity will be applied.
+
+    References
```

Comment:
Let's use a different reference; this was actually proposed in the original forget gate paper: http://cognet.mit.edu/system/cogfiles/journalpdfs/089976600300015015.pdf . Can we also move the references under the example?